frontend/survey-project/src/app/app.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["app.component.scss","app.component.css"],"names":[],"mappings":"AAAA;EACI,kBAAA;EACA,cAAA;EACA,WAAA;EACA,YAAA;ACCJ;;ADEA;EACI,kBAAA;ACCJ;;ADEA;EACI,eAAA;EACA,MAAA;EACA,OAAA;EACA,WAAA;EACA,YAAA;EACA,oCAAA;EACA,aAAA;EACA,mBAAA;EACA,uBAAA;EACA,aAAA;ACCJ","file":"app.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.